src/main/kotlin/no/digdir/accessrequestapi/model/DatasetMetadata.kt

```diff
@@ -60,6 +63,8 @@ data class DatasetMetadata(
                 ),
             language = language,
         )
+
+    fun isDatasetWithoutDistribution() = type == "datasets" && distribution.isNullOrEmpty()
```

Comment:
Dette er en metode internt i `DatasetMetadata`, vil ikke `type == "datasets"` alltid være true?
```suggestion
    fun isDatasetWithoutDistribution() = distribution.isNullOrEmpty()
```